src/schema/objects/metadata.yaml

```diff
@@ -1497,6 +1524,7 @@ MTPulseShape:
     `"SINC"`, `"SINCHANN"` (sinc pulse with Hanning window),
     `"SINCGAUSS"` (sinc pulse with Gaussian window), `"FERMI"`.
   type: string
+  # TODO: Add definitions for these values.
```

Comment:
What do the different `MTPulseShape` values mean?

Reply:
I think I figured it out. The descriptions aren't exactly extensive, but they're something.